JustSaying/MessagingConfig.cs

```diff
@@ -13,10 +16,12 @@ public MessagingConfig()
             PublishFailureBackoffMilliseconds = JustSayingConstants.DEFAULT_PUBLISHER_RETRY_INTERVAL;
             AdditionalSubscriberAccounts = new List<string>();
             Regions = new List<string>();
+            MessageResponseLogger = (r, m) => {};

```

Comment:
May as well leave it as `null` as it's always invoked with the null conditional operator